docs/build-docs.sh

```diff
@@ -82,4 +82,29 @@ fi
 export CDK_VERSION=$(../tools/pkgtools/bin/cdk-version)
 echo "Set CDK_VERSION=${CDK_VERSION} (consumed by conf.py)..."
 
+# Update tools.rst
+# First compile cdk.ts if cdk.js does not exist
+PATH=$PWD/../node_modules/.bin:$PATH
+echo "PATH:"
+echo $PATH
+if [ ! -d ../packages/aws-cdk/bin/cdk.js ]; then
```

Comment:
Sadly it's more complicated that that. The toolkit has dependencies that must be built first. Here are your options:

1. You could use the [`buildup`](https://github.com/awslabs/aws-cdk/blob/master/scripts/buildup) script if you like
2. Just print an error saying "you must first build aws-cdk in order for this to work"
3. Put placeholder text instead of the usage string in case the toolkit is not available (in the real build it will be available).